content/docs/manifest.json

```diff
@@ -671,38 +662,48 @@
             }
           ]
         },
-        {
-          "title": "CLI reference",
-          "routes": [
-            {
-              "title": "Introduction",
-              "path": "/docs/cli/README.md"
-            },
-            {
-              "title": "acmesolver",
-              "path": "/docs/cli/acmesolver.md"
-            },
-            {
-              "title": "cainjector",
-              "path": "/docs/cli/cainjector.md"
-            },
-            {
-              "title": "cmctl",
-              "path": "/docs/cli/cmctl.md"
+          {
+            "title": "Reference",
```

Comment:
This brings us a step closer to having a reference section like of other Kubernetes projects:
 * https://kubernetes.io/docs/reference/
 * https://linkerd.io/2.12/reference/
 * https://istio.io/latest/docs/reference/